packages/edit-site/src/components/global-styles/background-panel.js

```diff
@@ -16,7 +16,6 @@ const BACKGROUND_DEFAULT_VALUES = {
 const {
 	useGlobalStyle,
 	useGlobalSetting,
-	useGlobalStyleLinks,
```

Comment:
I like how the change in this PR means this part is happening internally to the panel now 👍